src/renderer/screens/market/DropDown.tsx

```diff
@@ -30,12 +30,16 @@ function DropdownControl<O, M extends boolean, G extends GroupBase<O>>(
 
 function DropdownValueContainer<O>(props: ValueContainerProps<O, false>) {
   const ChevronIcon = props.selectProps.menuIsOpen ? ChevronTopMedium : ChevronBottomMedium;
+  const { label } = (props.selectProps as unknown) as Props<O>;
 
   return (
     <ValueContainer
       {...props}
       render={() => (
         <FlexBox alignItems="center" flexDirection="row">
+          <Text fontWeight="semiBold" variant={"paragraph"} color="neutral.c80" mr={2}>
```

Comment:
not important at all but for the style nazi that I am, i'm curious why sometimes you usa string props like that `fontWeight="semiBold"` and sometimes like that `variant={"paragraph"}` haha

Reply:
(imo the first is cleaner)

Reply:
It's just my IDE :safety_vest:
I agree it's prettier yes but i think we don't care also ? The perfect solution would be a lint rule